onadata/apps/main/management/commands/create_enketo_express_urls.py

```diff
@@ -38,7 +39,7 @@ def handle(self, *args, **options):
             raise CommandError(
                 'please provide a server_name, a server_port and a protocol')
 
-        if server_name not in ['api.ona.io', 'stage-api.ona.io', 'localhost']:
+        if server_name not in APP_DOMAIN:
```

Comment:
Lets make `APP_DOMAIN` just a string. I don't believe there is a scenario we'll need more than one domain.

```suggestion
        if server_name == APP_DOMAIN:
```

_Actually if possible let's not limit the `server_name`... I think making it a required argument might be better ? Thoughts ?_